packages/hardhat/example.env

```diff
@@ -1,36 +0,0 @@
-# This is a template for the environment variables you'll need for 
-# deployment on Rinkeby and mainnet
```

Comment:
I actually like this pattern and suggest we don't delete this file.  Instead of having to search the app to find all environment variables used, it's common to check in an example file like this and add a step to the readme that says:

"Run `cp example.env .env` and add keys for the networks you'd like to use".